README.md

```diff
@@ -1,11 +1,26 @@
 # Melange
 
-A _mixture_ of tooling combined to produce JS from OCaml / Reason.
+### Tools for producing JS from Reason/OCaml.
 
-This project is a fork of the
-[ReScript compiler](https://github.com/rescript-lang/rescript-compiler/) with
-a focus on compatibility with the wider OCaml ecosystem. A small write-up with
-more details on the motivation behind this project can be found in this
+Melange is a compiler for producing performant Javascript from Reason and Ocaml.
+Powered by a strong static type system with best-in-class type inference based 
+on the OCaml compiler, Melange facilitates robust code.
+
+### Positioning in the broader OCaml ecosystem
+
+This project is forked from the
+[ReScript compiler](https://github.com/rescript-lang/rescript-compiler/) shifting
+focus on integration with the OCaml ecosystem. This enables code sharing between
+backend and frontend using Dune's virtual libraries.
```

Comment:
This is a bold claim. I've never tested virtual libraries with Melange, and I don't think they would be able to build a melange project.

Did you see an example where this is supported?

Reply:
Referenced here: https://jchavarri.github.io/melange-docs/

> Write your code once, share between native and browser apps using Dune virtual libraries

Reply:
Actually that _is_ possible. I remember Javier had a demo. I had forgotten!